galasa-ui/src/components/test-runs/OverviewTab.tsx

```diff
@@ -3,17 +3,41 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  */
-
-import React from "react";
+"use client";
+import React, { useEffect, useState } from "react";
 import styles from "@/styles/OverviewTab.module.css";
 import InlineText from "./InlineText";
 import { Tag } from "@carbon/react";
 import { RunMetadata } from "@/utils/interfaces";
 import { useTranslations } from "next-intl";
+import { Link } from "@carbon/react";
+import { Launch } from "@carbon/icons-react";
+import { getOneMonthAgo, getAWeekBeforeSubmittedTime } from "@/utils/timeOperations";
 
 const OverviewTab = ({ metadata }: { metadata: RunMetadata }) => {
   const tags = metadata?.tags || [];
   const translations = useTranslations("OverviewTab");
+  const [weekBefore, setWeekBefore] = useState<string | null>(null);
+  
+  const MONTH_AGO = getOneMonthAgo();
+
+  const OTHER_RECENT_RUNS = `/test-runs?testName=${metadata?.testName}&bundle=${metadata?.bundle}&group=${metadata?.group}&package=${metadata?.package}&from=${MONTH_AGO}`;
+  const RETRIES_FOR_THIS_TEST_RUN = `/test-runs?submissionId=${metadata?.submissionId}&from=${weekBefore}`;
```

Comment:
If we click on either of these links, we need to then manually clear the search criteria if we want to search for other runs, which can be cumbersome.

Wonder if we need a "Reset all"/"Clear all" button to clear out all search criteria in a single click